selfdrive/car/toyota/values.py

```diff
@@ -1152,6 +1157,23 @@ class CAR:
       b'\x028646F3304100\x00\x00\x00\x008646G2601200\x00\x00\x00\x00',
     ],
   },
+  CAR.LEXUS_ESH: {
```

Comment:
add CAR.LEXUS_ESH to IGNORED_FINGERPRINTS

Reply:
done!